router/internal/persistedoperation/operationstorage/cache.go

```diff
@@ -14,6 +12,23 @@ type OperationsCache struct {
 	Cache *ristretto.Cache[string, []byte]
 }
 
+func NewOperationsCache(cacheSize int64) (*OperationsCache, error) {
+	if cacheSize <= 0 {
+		return &OperationsCache{}, nil
+	}
+
+	cache, err := ristretto.NewCache(&ristretto.Config[string, []byte]{
+		NumCounters: cacheSize * 10,
+		MaxCost:     cacheSize,
+		BufferItems: 64,
```

Comment:
No need to have a cost function, since every `set` operation passes the appropriate cost along (the length of the document)